packages/ui-inputs/src/buttons.stories.tsx

```diff
@@ -232,30 +232,35 @@ ButtonDisabled.story = {
 export const Workshop: Story = () => {
     return <div className="row">
         <div className="col">
-            <Button label={text("Label", "Label")}
-                    isActive={boolean("Active", false)}
-                    disabled={boolean("Disabled", false)}
-                    type={select(typeLabel, typeOptions, typeDefaultValue) as 'default' | 'cta' | 'borderless'}
-                    layout={select(layoutLabel, layoutOptions, 'horizontal') as 'horizontal' | 'vertical'}
-                    onClick={action('clicked')}></Button>
+            <Button
+                isProcessing={false}
```

Comment:
you should use here a knob so it will be interactive for the workshop
```
isProcessing={boolean("Is Processing", false)}
```

same in the other button